server/controllers/CommentController.js

```diff
@@ -0,0 +1,87 @@
+// module import
+import model from '../db/models';
+
+
+const { Comment } = model;
+/**
+ * @class CommentController
+ */
+class CommentController {
+  /**
+   * @static
+   * @description The create comment method
+   * @param  {object} request The request object
+   * @param  {object} response The response object
+   * @returns {object} json response
+   * @memberof CommentController
+   */
+  static addCommentToArticle(request, response) {
+    const user = request.userObject.dataValues;
+    const article = request.articleObject.dataValues;
+    // get the user id
+    const userId = user.id;
+    // get the article id
+    const articleId = article.id;
+    // get content of the request
+    const { commentBody } = request.body;
+
+    const commentObject = {
+      userId,
+      articleId,
+      commentBody
+    };
+
+    Comment.create(commentObject)
+      .then((comment) => {
+        if (!comment) {
+          return response.status(500).json({
+            message: 'Internal Server Error',
+          });
+        }
+        return response.status(201)
+          .json({
+            message: 'Comment Added Successfully',
+            data: { comment, user }
+          });
+      })
+      .catch(err => response.status(400).json({ message: err.message }));
```

Comment:
Can we not make it a one-liner,  @koiic, please look through it.
 It should follow this style, check all occurrences.
```js 
  .catch(err => response.status(400)
         .json({
            message: err.message 
        }));
```